### DISCUSSION TITLE
Getting harder and harder to just get a react page up

### DISCUSSION THREAD

**Discussion by nick-tw** [vercel/vercel#6531] [Ideas] [answered]
Vercel (Zeit when I used it) used to be so easy to get a react page up. I'd click on a template, clone, add some code, and push. It'll be up.

NOW, I click on a create react app template, clone. 
Push. 
Errors out. Realize I have the wrong identity, so I change my git email for the commits. 
Repush.
Errors out. Apparently there's some build setting I need to override the build command to add "CI= npm build"
Repush.
Errors out. Apparently I need to define a build output setting...? I override.
Repush.
Still errors out. Getting "Error: No Output Directory named "build" found after the Build completed. You can configure the Output Directory in your Project Settings." I check the docs and have performed those steps.

Now I'm stuck.

Guys, I paid for your service to make this as easy as clicking a button and pushing to a repo. The template I click on your site that generates a react page should work out of the box, but it's not. This is way too much setup.

**Comment by leerob** [comment, marked as answer]
To confirm, if you deploy the [Create React App template](https://vercel.com/new/clone?s=https%3A%2F%2Fgithub.com%2Fvercel%2Fvercel%2Ftree%2Fmain%2Fexamples%2Fcreate-react-app&template=create-react-app&id=67753070&b=main&from=templates), you're seeing errors? Could you post them here?

**Reply by nick-tw** [reply]
I just created a new repo using the create react and it seems to be fixed. Will close.
